packages/backend/src/app/pieces/piece-metadata-service/cloud-piece-metadata-service.ts

```diff
@@ -0,0 +1,51 @@
+import { PieceMetadataSchema } from '../piece-metadata-entity'
+import { GetParams, PieceMetadataService } from './piece-metadata-service'
+import { PieceMetadata, PieceMetadataSummary } from '@activepieces/pieces-framework'
+import { AllPiecesStats, pieceStatsService } from './piece-stats-service'
+import { StatusCodes } from 'http-status-codes'
+import { ActivepiecesError, ErrorCode } from '@activepieces/shared'
+
+const CLOUD_API_URL = 'https://activepieces-cdn.fra1.digitaloceanspaces.com/pieces/metadata'
+
+const handleHttpErrors = async (response: Response) => {
+    if (response.status === StatusCodes.NOT_FOUND) {
+        throw new ActivepiecesError({
+            code: ErrorCode.ENTITY_NOT_FOUND,
+            params: {
+                message: 'piece not found',
+            },
+        })
+    }
+
+    if (response.status !== StatusCodes.OK) {
+        throw new Error(await response.text())
+    }
+}
+
+export const CloudPieceMetadataService = (): PieceMetadataService => {
+    return {
+        async list(): Promise<PieceMetadataSummary[]> {
+            const response = await fetch(`${CLOUD_API_URL}/latest.json`)
```

Comment:
I assume this will be changed once it's ready, just resolve this comment if that is correct

Reply:
yup correct, so that things don't break after merging